src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -58,6 +59,7 @@ internal static void ForbidHtmlCharacters(AllowedCharactersBitmap allowedCharact
         public DefaultHtmlEncoder(params UnicodeRange[] allowedRanges) : this(new CodePointFilter(allowedRanges))
         { }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
If this really is to be internal, why add this attribute? VS at least historically ignores this attribute if you're within the solution that contains this type.